modules/installation-cloudformation-worker.adoc

```diff
@@ -17,3 +16,23 @@ that you need for your {product-title} cluster.
 include::https://raw.githubusercontent.com/openshift/installer/release-4.18/upi/aws/cloudformation/06_cluster_worker_node.yaml[]
 ----
 ====
+[IMPORTANT]
+====
+When you use the CloudFormation template for the control plane machines, it provisions all three control plane machines with a single stack. However, when you use the CloudFormation template to deploy the worker machines, you must create the number of stacks based on the number that you defined in the `install-config.yaml` file. Each stack is provisioned one time only for each machine. To provision a new worker machine, you must change the stack name.
+====
+
+[source,terminal]
```

Comment:
This command could do with a lead-in sentence as the context of the command dangles with little context after a big configuration block.